Formula/gitlab-gem.rb

```diff
@@ -0,0 +1,33 @@
+class GitlabGem < Formula
+  desc "Ruby client and CLI for GitLab API"
+  homepage "https://github.com/NARKOZ/gitlab"
+  url "https://github.com/NARKOZ/gitlab/archive/v4.3.0.tar.gz"
+  sha256 "b2679d0696642f67ba84a4f9df21ad4a7bf1bc814d7f86218b72ebe5e56ff2d0"
+
+  head "https://github.com/NARKOZ/gitlab.git"
+
+  bottle :unneeded
```

Comment:
This isn't the case.

Reply:
Can you point me how to get checksums for different macOS versions?

Reply:
CI will handle it for you. You don't need to do anything beside remove this line.